src/Module/Login.php

```diff
@@ -181,8 +181,7 @@ public static function sessionAuth()
 					intval($_SESSION['visitor_id'])
 				);
 				if (DBM::is_result($r)) {
-					$a = self::getApp();
-					$a->contact = $r[0];
+					self::getApp()->contact;

```

Comment:
You left the variable assignment out, I’ll be more careful with my comment next time 😅

Reply:
ohhh embarrassing.